src/person/views.py

```diff
@@ -59,15 +59,19 @@ def delete(self, request):
 
 
 class UserView(generics.GenericAPIView):
-    serializer_class = UserSerializer
+    serializer_class = SimpleUserSerializer
     permission_classes = api_settings.CONSUMER_PERMISSIONS
 
     def get(self, request, id):
         """Get user by id."""
         user = User.objects.filter(id=id)
         if not user:
             return failure_response("User not found.", status.HTTP_404_NOT_FOUND)
-        return success_response(self.serializer_class(user[0]).data, status.HTTP_200_OK)
+        serialized_user = self.serializer_class(
+            user[0], context={"request_user": request.user}
```

Comment:
i feel like you usually send in the entire request for the context but i guess this is ok